packages/slice-machine/src/apiClient.ts

```diff
@@ -17,6 +18,12 @@ const defaultAxiosConfig = {
   },
 };
 
+/** State Routes **/
+
+export const getState = (): Promise<AxiosResponse<ServerState>> => {
+  return axios.get("/api/state", defaultAxiosConfig);
```

Comment:
you can pass `ServerState` as a type parameter to `axios.get` not sure if it makes a difference here, as the response object is kept

Reply:
The type we put in the `axios.get` function is only use on `then`, `catch`etc ... functions, but doesn't help typescript to infer the return type of the function, so now I put both :)